chall02/vtran.c

```diff
@@ -0,0 +1,27 @@
+#include <string.h>
+#include <stdlib.h>
+
+static char*    light_itoa_base(int nb, int base)
+{
+    char str[17];
+    char ret[3];
+
+    strcpy(str, "0123456789abcdef");
```

Comment:
you could have declared str directly since you don't have to respect the school norm :)